x-pack/plugins/observability_solution/infra/public/pages/metrics/inventory_view/components/waffle/group_name.tsx

```diff
@@ -71,29 +70,31 @@ interface InnerProps {
   isChild?: boolean;
 }
 
-const Inner = euiStyled.div<InnerProps>`
-  border: 1px solid ${(props) => props.theme.eui.euiBorderColor};
+const Inner = styled.div<InnerProps>`
+  border: 1px solid ${(props) => props.theme.euiTheme.border.color};
```

Comment:
Suggestion: This could be simplified using `euiTheme.border.thin`. (same for `Count` and other following styled components)

```tsx
border: ${props.theme.euiTheme.border.thin};
```